examples/mandelbrot-golem-cluster.yaml

```diff
@@ -94,9 +94,9 @@ worker_start_ray_commands: [
 ]
 
 # Authentication credentials that Ray will use to launch nodes
-auth:
-  # TODO: try to remove explicit ssh_user field
-  ssh_user: 'root'
+# auth:
+  # Custom username for ssh
+  # ssh_user: "root"
```

Comment:
Do we want to keep this here?

Reply:
This change was done only for the sake of keeping full example up to date with top level `golem-cluster.yaml`.